pkg/client/didexchange/client.go

```diff
@@ -133,11 +136,17 @@ func (c *Client) QueryConnections(request *QueryConnectionsParams) ([]*QueryConn
 	}, nil
 }
 
-// QueryConnectionByID fetches single connection record for given id
-func (c *Client) QueryConnectionByID(id string) (*QueryConnectionResult, error) {
-	// TODO sample response, to be implemented as part of #226
+// GetConnection fetches single connection record for given id
+func (c *Client) GetConnection(connectionID string) (*QueryConnectionResult, error) {
+	conn, err := c.recorder.GetConnection(connectionID)
```

Comment:
`recorder` seems to be used as a `connectionStore`.

Reply:
Rename to connectionStore